packages/arb-token-bridge-ui/tests/e2e/specs/switchNetworks.cy.ts

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
Kudo for writing test! If we could add test for URL query param changes, that would be even more awesome!